flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
an ebook reader that doesn't run on aarch64?

Reply:
Right, I forgot I had this file. Should I completely remove this, or should I just add aarch64 to the list?